Modules/Microsoft365DSC/DSCResources/MSFT_IntuneDeviceManagmentAndroidDeviceOwnerEnrollmentProfile/MSFT_IntuneDeviceManagmentAndroidDeviceOwnerEnrollmentProfile.psm1

```diff
@@ -0,0 +1,699 @@
+function Get-TargetResource
+{
+    [CmdletBinding()]
+    [OutputType([System.Collections.Hashtable])]
+    param
+    (
+        [Parameter()]
+        [System.String]
+        $Id,
+
+        [Parameter(Mandatory = $true)]
+        [System.String]
+        $DisplayName,
+
+        [Parameter()]
+        [System.String]
+        $AccountId,
+
+        [Parameter()]
+        [System.Boolean]
+        $ConfigureWifi,
+
+        [Parameter()]
+        [System.DateTime]
+        $CreatedDateTime,
```

Comment:
Let's remove since it cannot be configured. Only CRUD enabled properties should be included in DSC resources.